vistautils/parameters.py

```diff
@@ -820,8 +808,10 @@ def _inner_load(
                 for included_file in raw_yaml["_includes"]:
                     _logger.info("Processing included parameter file %s", included_file)
                     if not os.path.isabs(included_file):
-                        if root_path is not None:
-                            included_file_path = Path(root_path, included_file).resolve()
+                        if includes_are_relative_to is not None:
+                            included_file_path = Path(
+                                includes_are_relative_to, included_file
+                            ).resolve()
```

Comment:
```suggestion
                            ).resolve(strict=True)
```
to make sure an exception gets raised if the path doesn't exist.